packages/web-components/components/snackbar.ts

```diff
@@ -0,0 +1,8 @@
+// import { defineCustomElement } from 'vue';
+// import { PuikSnackbar } from '@prestashopcorp/puik-components';
+// import type { CustomElementWithTagName } from '../types'
+
+// const PuikSnackbarCe = defineCustomElement(PuikSnackbar) as unknown as CustomElementWithTagName;
+// PuikSnackbarCe.tagName = 'puik-snackbar-ce';
+
+// export default PuikSnackbarCe;
```

Comment:
Why is it comment ?

Reply:
the snackbar is the only component that screws up the build at the moment.. that's why I commented on it (I think we should see this problem when we check the web-components one by one )

Reply:
OK, I understand